packages/icondusk-e2e/package.py

```diff
@@ -18,16 +18,15 @@ class IconduskE2e(CMakePackage):
     maintainers = ['cosunae']
 
     version('master', branch='master')
-    version('0.1-dev', tag='0.1-dev')
+    version('dev-build', branch='master')
 
     depends_on('cmake@3.17.0:')
     depends_on('boost@1.73.0%gcc')
     depends_on('netcdf-fortran')
     depends_on('netcdf-c')
     depends_on('atlas_utilities', type=('build', 'run'))
     depends_on('dawn4py',  type=('build'))
-    depends_on('dusk',  type=('build'))
-    depends_on('dusk@icondusk-e2e',  type=('build'), when='@0.1-dev')
+    depends_on('dusk@icondusk-e2e',  type=('build'), when='@master')
```

Comment:
shouldn't also the `dev-build` version depend on `dusk@icondusk-e2e`? if the user then wants to override the dusk version it still can be done in the `dev-build` command's spec

Reply:
yes, done